recipes/evalhyd-python/build.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+set -ex
+
+${PYTHON} -m pip install . -vvv
```

Comment:
Why not use the `script` entry in the meta.yaml and remove both build and bld files?

Reply:
Yes, you are right, that's an overkill. I will fix this.